### ISSUE TITLE
Germs.io Small Party / Dream Party

### ISSUE THREAD

**Issue by PoP-Saaske** [Stasadance/Germs.io#6]
Hello, I'm Saaske I come with this idea: so that the germs.io, as we know today, will be before again. I would like them to put in '' party '' the game mode that I like, as before, or put a dream party with many viruses and increase of mini cells, and putting the map smaller, as before. As I will put in this photo: 

https://z-p3-scontent.fccs1-1.fna.fbcdn.net/v/t35.0-12/29884277_610750875938208_798695467_o.png?_nc_cat=0&oh=ff0c7fe714d17909d4667094913b7522&oe=5AC508C7

Excuse me if I speak English badly, I use translator = (
In short: put the dream party mode '' as it was before the update 3.4 '' or return to the old mode without put another mode, only return to the mode as it was before party =), Thank you for your attention, A greeting.-Saaske

**Comment by Stasadance** [comment]
Do you mean the border size, and how many viruses? Those are the only things i changed in v3.4

**Comment by PoP-Saaske** [comment]
i mean broder size
do you can't anything??!?!
